pyinteraph/path_analysis.py

```diff
@@ -482,6 +482,32 @@ def reorder_graph(graph, identifiers):
     ordered_graph.add_weighted_edges_from(weighted_edges)
     return(ordered_graph)
 
+def write_metapath_table(graph, identifiers, residue_names, fname):
+        '''The function returns a file csv containing the information
+        about the edges of the metapath graph and their weight.'''
+
+        # Create dictionary of {ids : resnames}
+        nodes_dict = dict(zip(identifiers, residue_names))
+        
+        with open(fname, 'w') as out:
+            
+            # Write header
+            out.write('chain1,resid1,restype1,group1,' \
+                      'chain2,resid2,restype2,group2,weight \n')
+
+            # Define chain, resid, and resname for each node 
+            # of each edge, and the edge weight
+            for node1, node2, w in graph.edges(data=True):
+                chain1, chain2 = node1[0], node2[0]
+                resid1, resid2 = int(''.join([x for x in list(node1) if x.isdigit()])), \
+                                 int(''.join([x for x in list(node2) if x.isdigit()]))
+                restype1, restype2 = nodes_dict[node1], nodes_dict[node2]
+                e_weight = w['weight']*100
```

Comment:
I see why you did this here - but then I think this should be done when we normalise the graph (i.e. it should be in the `normalize_graph` function), so that the values we print in the csv and those in the plots are consistent. If we want to add this, it should be done in a separate Issue/PR to be taken after this one, so I would remove it from here